index.html

```diff
@@ -81,7 +81,7 @@ <h3>
 
 
 <blockquote>
-<p><img class="emoji" title=":bulb:" alt=":bulb:" src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f4a1.png" height="20" width="20" align="absmiddle"><strong>Tip:</strong>
+<p><img class="emoji" title=":bulb:" alt=":bulb:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f4a1.png" height="20" width="20" align="absmiddle"><strong>Tip:</strong>
```

Comment:
@yousseb i was about to send this PR but saw it's already been fixed in this one.  why not accept it?

this is what it fixes:

<img width="811" alt="Screenshot 2023-05-11 at 7 09 22 PM" src="https://github.com/yousseb/meld/assets/251287/b32a2389-ad89-4b78-b6c0-974bac19bf20">